src/utility/svd.jl

```diff
@@ -34,20 +36,95 @@ SVD from `KrylovKit.svdsolve` is used.
 """
 PEPSKit.tsvd(t, alg; kwargs...) = PEPSKit.tsvd!(copy(t), alg; kwargs...)
 function PEPSKit.tsvd!(t, alg::SVDAdjoint; trunc::TruncationScheme=notrunc(), p::Real=2)
-    return TensorKit.tsvd!(t; alg=alg.fwd_alg, trunc, p)
+    return TensorKit.tsvd!(t; alg, trunc, p)
```

Comment:
```suggestion
    return TensorKit.tsvd!(t; alg.fwd_alg, trunc, p)
```
Probably this still needs to unpack the `fwd_alg` before deferring to `TensorKit.tsvd!`?